examples/broken-link-callbacks.rs

```diff
@@ -0,0 +1,37 @@
+extern crate pulldown_cmark;
+
+use pulldown_cmark::{html, BrokenLink, Options, Parser};
+
+fn main() {
+    let input: &str = "Hello world, check out [my website][].";
+    println!("Parsing the following markdown string:\n{}", input);
+
+    // Setup callback that sets the URL and title when it encounters
+    // a reference to our home page.
+    let callback = &mut |broken_link: BrokenLink| {
+        if broken_link.reference.as_ref() == "my website" {
+            println!(
+                "Replacing the markdown `{}` of type {:?} with a working link",
+                &input[broken_link.span], broken_link.link_type,
+            );
+            Some(("http;//example.com".into(), "my example website".into()))
```

Comment:
```suggestion
            Some(("http://example.com".into(), "my example website".into()))
```

Reply:
This seems like it should go in, I think.

Reply:
Done.